website/templates/project/project.mako

```diff
@@ -256,11 +256,11 @@
             <div class="panel-body" style="display:none">
                 <dl id="citationList" class="citation-list">
                     <dt>APA</dt>
-                        <dd class="citation-text" data-bind="text: apa"></dd>
+                        <dd class="citation-text" data-bind="html: apa"></dd>
```

Comment:
I think the normal practice will be trying not to use html bind. @abought thought?